crates/re_types/definitions/rerun/blueprint/components/entities_determined_by_user.fbs

```diff
@@ -11,9 +11,11 @@ namespace rerun.blueprint.components;
 
 /// Whether the space view entities were manually edited.
 table EntitiesDeterminedByUser (
+    "attr.arrow.transparent",
     "attr.rerun.scope": "blueprint",
     "attr.rust.derive": "Default, PartialEq, Eq, PartialOrd, Ord",
-    "attr.rust.repr": "transparent"
+    "attr.rust.repr": "transparent",
+    "attr.rust.tuple_struct"
```

Comment:
I'm surprised we don't have a python alias defined here, I got mypy errors when I didn't put `bool` in there explicitely

Reply:
I think this is something that come over partially from the cherry-pick